README.md

```diff
@@ -125,7 +125,7 @@ This project is set up to use [jest](https://facebook.github.io/jest/) for tests
 This repository uses Appium to run UI tests. The tests live in `__device-tests__` and are written using Appium to run tests against simulators and real devices. To run these you'll need to check off a few things: 
 
 * When running the tests, you'll need to ensure the Metro bundler (`npm run start`) is not running. 
-* [Appium CLI](https://appium.io/docs/en/about-appium/getting-started/) installed and available globally. We also recommend using [appium-doctor](https://github.com/appium/appium-doctor) to ensure all of Appium's dependencies are good to go. You don't have to worry about starting the server yourself, the tests handle starting the server on port 4723, just be sure that the port is free or feel free to change the port number in the test file. 
```

Comment:
Not sure where this change is coming from, at least I can't find the commit that introduced it 🤔. In any case, the old link is the correct one as `https://github.com/appium/appium/blob/master/docs/en/about-appium/getting-started.md` doesn't exist. I'll open a PR to fix it 💨 .

Reply:
@fluiddot This change initially came from [this PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5213) where I found a dead Appium link (github.com) while doing some testing. I updated it to the appium.io link, but it appears the change may have been missed in a merge. I've re-opened [another PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5228) with the correct link, but it is not urgent for 1.84.1 if it complicates things, as it has been there for a while.

Reply:
Ah, ok, thanks for clarifying it @derekblank, and opening a PR to address the change 🙇 .

> I've re-opened https://github.com/wordpress-mobile/gutenberg-mobile/pull/5228 with the correct link, but it is not urgent for 1.84.1 if it complicates things, as it has been there for a while.

Sure, this has no impact on the release so no rush on fixing it.